models/Element/Recyclebin/Item.php

```diff
@@ -400,7 +400,7 @@ public function unmarshalData($data)
         $copier->addTypeFilter(
             new \DeepCopy\TypeFilter\ReplaceFilter(
                 function ($currentValue) {
-                    if ($currentValue instanceof Model\Version\ElementDescriptor) {
+                    if ($currentValue instanceof Element\ElementDescriptor) {
```

Comment:
when you have already deleted data with `Model\Version\ElementDescriptor` relations? it will be ignored when restoring from recyclebin, right?

Reply:
did you see the alias ?

Reply:
yes, you are right! it should work with alias. 😜